### PR TITLE
Updated News Details Screen Layout

### LINKED ISSUE #9: Improve UI of headlines detail
The UI of headline details can be improved by improving the text style and color. Also in the grey space below the picture the news source and time of publishing of articles has to be added.

### PR DESCRIPTION
**Pull request by thedevelopersanjeev**
Fixes #9

### CODE REVIEW

File: app/src/main/res/layout/fragment_headline.xml [hunk 1]
```diff
@@ -16,6 +16,16 @@
...
+            android:layout_height="match_parent" />
+        </android.support.v4.widget.SwipeRefreshLayout>
+
+        <ProgressBar
+            android:id="@+id/progressBar"
+            android:layout_width="40dp"
+            android:layout_height="40dp"
+            android:layout_gravity="center"
+            android:visibility="visible" />
+    </FrameLayout>
```
Comment (sridharjajoo) [inline app/src/main/res/layout/fragment_headline.xml]: what's this? How can frame layout  be closed here. You can clearly see that the below lines aren't present inside any of the tags

File: app/src/main/java/com/example/sridharjajoo/newsapp/core/Headline/NewsDetailActivity.java [hunk 2]
```diff
@@ -23,6 +23,9 @@
 
     private ActivityNewsDetailBinding binding;
 
+    @BindView(R.id.tv_time)
+    TextView newsTime;
```
Comment (sridharjajoo) [inline app/src/main/java/com/example/sridharjajoo/newsapp/core/Headline/NewsDetailActivity.java]: Why is this required? It isn't used anywhere

File: app/src/main/java/com/example/sridharjajoo/newsapp/core/Headline/HeadlineAdapter.java [hunk 3]
```diff
@@ -127,6 +127,10 @@ private void configureViewHolder(HeadlineViewHolder viewHolder, Articles current
         viewHolder.description.setText(currentItem.title);
         viewHolder.newsSource.setText(currentItem.source.name);
         viewHolder.shareButton.setOnClickListener(view -> {
+            Intent intent = new Intent(Intent.ACTION_SEND);
+            intent.setType("text/plain");
+            intent.putExtra(Intent.EXTRA_TEXT, currentItem.url);
+            context.startActivity(Intent.createChooser(intent, "Share News"));
```
Comment (sridharjajoo) [inline app/src/main/java/com/example/sridharjajoo/newsapp/core/Headline/HeadlineAdapter.java]: This feature is already present in the below line. Why add it once again?

**Comment by sridharjajoo** [comment]
And please attach the screenshots as well.

File: app/src/main/java/com/example/sridharjajoo/newsapp/core/Headline/HeadlineAdapter.java (hunk 3 above)
Comment (thedevelopersanjeev) [reply app/src/main/java/com/example/sridharjajoo/newsapp/core/Headline/HeadlineAdapter.java]: removed those lines... these were added by mistake during resolving merge conflicts.

**Comment by sridharjajoo** [comment]
The design looks fine. It would be great of you add the date of publishing of the event just above the news source. In that way the details would be more descriptive.

**Comment by thedevelopersanjeev** [comment]
@sridharjajoo 

screenshot_20181223-215740
